app/assets/javascripts/validation.coffee

```diff
@@ -358,6 +357,21 @@ $(document).ready ->
     error.id = id
     error.element = $("##{id}")
 
+
+    if error.dataPath.split('/')[error.dataPath.split('/').length - 1] == 'AllowMultipleValues'
+      # Allow Multiple Values fields, under Subset, are boolean fields represented
+      # by radio buttons
+      # for a required error, because these are radio buttons, we want the error
+      # to be surfaced with the radio parent container, which the path
+      # transformation has taken care of
+      if error.message == 'should be boolean'
+        # html cannot hold boolean values, they are represented as string
+        # TODO: we can check the values to verify they are 'true' and 'false'.
```

Comment:
This feels excessive to me because I'm not sure what scenario leads to this value not being true or false.

Reply:
OTOH, it shouldn't be burdensome to check...